vue/components/ui/molecules/input-image/input-image.vue

```diff
@@ -1,7 +1,6 @@
 <template>
     <upload-area
         class="input-image"
-        v-bind:files.sync="files"

```

Comment:
Why?

Reply:
See the Issue section of this PR.
Basically, the tests warned of this dead code. This `sync` does nothing, as isn't referenced anywhere in the code + if you check [this line](https://github.com/ripe-tech/ripe-components-vue/blob/a52cb35274b64dbcf8cba9d2ea9f82c40b7accc7/vue/components/ui/molecules/input-image/input-image.vue#L6 it seems to handle the files update event.